doc/index.rst

```diff
@@ -10,18 +10,25 @@ Controller for the Simonyi Survey Telescope dome at Vera C. Rubin Observatory.
 
 .. _lsst.ts.Dome-using:
 
-Using lsst.ts.Dome
+Using lsst.ts.salobj
```

Comment:
Why did you change this to salobj?

Reply:
Ouch, copy and paste error. Nice catch and sorry about that!